lib/tty/file.rb

```diff
@@ -288,16 +288,26 @@ def create_file(relative_path, *args, context: nil, force: false, skip: false,
     # @api public
     def copy_file(source_path, *args, context: nil, force: false, skip: false,
                   verbose: true, color: :green, noop: false, preserve: nil, &block)
+
       source_path = source_path.to_s
+      unless ::File.file? source_path
+        log_status(:error, source_path, verbose: verbose, color: :red)
+        return
+      end
```

Comment:
I believe this statement makes the directory check unreachable. Also, I much prefer to allow copying a file to a directory location rather than providing an error. Wouldn't you agree?